x-pack/legacy/plugins/apm/public/components/shared/Links/apm/ErrorDetailLink.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { APMLink, APMLinkExtendProps } from './APMLink';
+
+interface Props extends APMLinkExtendProps {
+  serviceName: string;
+  errorGroupId: string;
+}
```

Comment:
Maybe it's just the name of `APMLinkExtendProps` that throws me off but might be more clear to do:

```
interface Props extends EuiLinkAnchorProps {
  serviceName: string;
  errorGroupId: string;
  children?: React.ReactNode;
}
```

Reply:
APMLinkExtendProps explicitly disallows `path` and `query` to prevent prop conflicts (e.g. `serviceName` and `errorGroupId` are used to build a path, and `uiFilters` are used to build a query). Would be happy with a better name though, if you have any suggestions 😅